nixos/default.nix

```
@@ -33,6 +42,15 @@ in
         <option>users.users.&lt;name?&gt;.packages</option> option.
       '';
 
+      backupExt = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        example = "backup";
+        description = "On activation move existing files to new path rather than fail.";
```

A bit long, can split it a bit
```suggestion
        description = ''
          On activation move existing files to new path rather than fail.
        '';
```